jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -75,19 +75,33 @@
           ></v-select>
         </v-row>
 
+        <v-row v-if="trace_type_selected!=='Flat'">
+          <v-text-field
+            label="Order"
+            type="number"
+            v-model.number="trace_order"
+            :rules="[() => trace_order!=='' || 'This field is required',
+                     () => trace_order>=0 || 'Order must be positive',
+                     () => (trace_type_selected!=='Spline' || (trace_order > 0 && trace_order <= 5)) || 'Spline order must be between 1 and 5']"
+            hint="Order of the trace model."
+            persistent-hint
+          >
+          </v-text-field>
+        </v-row>
+
         <v-row>
           <v-text-field
             label="Pixel"
             type="number"
             v-model.number="trace_pixel"
             :rules="[() => trace_pixel!=='' || 'This field is required']"
-            :hint="trace_type_selected === 'Flat' ? 'Pixel row for flat trace.' : 'Pixel row initial guess for auto trace.'"
+            :hint="trace_type_selected === 'Flat' ? 'Pixel row for flat trace.' : 'Pixel row initial guess for fitting the trace.'"
```

Comment:
@bmorris3 - does this make sense to you given the conversation in https://github.com/spacetelescope/jdaviz/pull/1827#discussion_r1016686961?

Reply:
Looks great. 👌🏻